### PR TITLE
Fix nil log panic

### PR DESCRIPTION
**Pull request by gabriel**
https://ci.keybase.io/job/client/job/master/159/execution/node/167/log/

### CODE REVIEW

**Comment by gabriel** [comment]
@maxtaco

File: go/logger/standard.go [hunk 1]
```diff
@@ -355,37 +355,37 @@ func (log *Standard) GetUnforwardedLogger() *UnforwardedLogger {
 }
 
 func (log *UnforwardedLogger) Debug(s string, args ...interface{}) {
-	if log.internal != nil {
+	if log != nil && log.internal != nil {
```
Comment (maxtaco) [inline go/logger/standard.go]: is the original problem that `log` is `nil` and not that `log.internal` is `nil`?

Comment (gabriel) [inline go/logger/standard.go]: yeah I think so

Comment (maxtaco) [inline go/logger/standard.go]: hmm well that helps us track it down a little bit better. @songgao was going to look into it a little bit further.

**Comment by maxtaco** [comment]
Let's 👍  for now, but I think we should hunt down the deeper cause.

**Comment by gabriel** [comment]
🎉 https://keybase.atlassian.net/browse/CORE-3452

**Pull request merged**